rubygem-concurrent-ruby-edge/rubygem-concurrent-ruby-edge.spec

```diff
@@ -16,8 +16,7 @@ Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) >= 1.0.0
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) < 1.1.0
+Requires: %{?scl_prefix}rubygem(concurrent-ruby) = 1.0.3
```

Comment:
@domcleal Changed this to rely on our packages instead of the SCL - I don't mind submitting changes to the SCL but I couldn't figure out where? I suppose https://github.com/domcleal/sclo-ror42 is not the most updated version of the SCL